__tests__/intergration.test.js

```diff
@@ -161,6 +160,25 @@ describe("GET /api/articles/", () => {
             expect(resultArray).toBeSorted({ key: 'created_at', descending: true });
         });
     })
+    test("Should return bad request if topic query is anything other than a string", () => {
+        return request(app).get("/api/articles?topic=123").expect(400).then(result => {
+            expect(result.body.message).toBe("Bad request")
+        })
+    })
+    test("Should return only articles that match if passed a valid topic query", () => {
+        return request(app).get("/api/articles?topic=cats").expect(200).then(result => {
+            expect(result.body.articles.length).toBe(1)
+        })
+    })
+    test("all returned objects should have the topic of matching query", () => {
+        return request(app).get("/api/articles?topic=mitch").expect(200).then(result => {
+            const articles = result.body.articles;
+            for (const article of articles) {
+                expect(article.topic).toBe("mitch");
+            }
+        })
+    })
+
```

Comment:
Good testing here, but we want to make sure we're passing them for the right reason.